tests/nightly/test_large_array.py

```diff
@@ -1168,6 +1169,80 @@ def check_correctness(mxnet_op, numpy_op, atol=1e-3):
     check_binary_broadcast()
 
 
+def test_linalg():
+    def get_large_identity_mat():
+        A = nd.zeros((LARGE_SQ_X, LARGE_SQ_X))
+        for i in range(LARGE_SQ_X):
+            A[i,i] = 1
+        return A
+
+    def batchify(mat):
```

Comment:
i didn't understand the purpose does it make yoour tensor (x,x) -> (1,x,x)

Reply:
yes, this is for invoking the batch version of the same operator

Reply:
Atleast run it 2 times. Make it (2,x,x) then

Reply:
an alternative would be to test (1,large,large) plus (2,small,small). I still think testing (2,large,large) is too time consuming

Reply:
yes

Reply:
@access2rohit checking values is not the time consuming step here. We will hardly save time by just checking for a single value. The major time taking step is the operator itself, which runs twice for the same input in case of (2,x,x).

Reply:
@mseth10 That's the whole point, since we are testing for Large Tensors. @Zha0q1 (2, small, small) is not our use case to be tested in the first place

Reply:
added batch test for (2,x,x)